cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -45,7 +41,7 @@ This defense is one of the most popular and recommended methods to mitigate CSRF
 
 ### Synchronizer Token Pattern
 
-Any state changing operation requires a secure random token (e.g., CSRF token) to prevent CSRF attacks. A CSRF token should be unique per user session, large random value, and also generated by a cryptographically secure random number generator. The CSRF token is added as a hidden field for forms, headers/parameters for AJAX calls (It is recommended to add in parameter than in header. If you need to add it to header, it is better to make sure that the token header is not being logged at your server) and within the URL if the state changing operation occurs via a GET. See "Disclosure of Token in URL" section below. The server rejects the requested action if the CSRF token fails validation.
+Any state changing operation requires a secure random token (e.g., CSRF token) to prevent CSRF attacks. A CSRF token should be unique per user session, a large random value, and also generated by a cryptographically secure random number generator. The CSRF token is added as a hidden field for forms, headers/parameters for AJAX calls (If you need to add it to header, it is better to make sure that the token header is not being logged at your server) and within the URL if the state changing operation occurs via a GET. See "Disclosure of Token in URL" section below. The server rejects the requested action if the CSRF token fails validation.
```

Comment:
1. Why are we saying that most practices implement CSRF in a hidden field? We can simply mention the techniques (in the headers, hidden fields, etc.) to do and where they can be used (forms, AJAX calls, etc.)
2. Instead of saying `See Disclosure of Token ...`, directly link to the section

Reply:
@ThunderSon I am not sure if I got your first point. Where did we mention that "most practices implement CSRF in a hidden field"?? We mentioned "The CSRF token is added as a hidden field for forms, headers/parameters for AJAX calls"

Reply:
Exactly. Who said that? We can simply mention that CSRF token can be added through hidden fields, headers, additional parameters in forms, AJAX calls, etc. Just mention all possible techniques to add the token for all state changing actions

Reply:
done

Reply:
I have not seen many code samples where people include tokens in headers with form submissions and hence suggested to mention it considering the regular trend. Anyway, I am fine with the way its edited now.